server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -34,7 +33,7 @@ def process(self, instance):
         yeti_nodes = cmds.ls(instance[:], type="pgYetiMaya", long=True)
         for node in yeti_nodes:
             # Get Yeti resources (textures)
-            resources = self.get_yeti_resources(node)
+            resources = self.get_texture_resources(node)
```

Comment:
Seems like this may need to be reverted because the method is still called `get_yeti_resources`?
```suggestion
            resources = self.get_yeti_resources(node)
```